common/src/main/res/layout/app_bar_main.xml

```diff
@@ -1,49 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<com.google.android.material.appbar.AppBarLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/appbar"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:background="@android:color/transparent"
-    android:theme="@style/My.AppBarOverlay"
-    app:elevation="0dp">
-
-    <androidx.appcompat.widget.Toolbar
-        android:id="@+id/toolbar"
-        android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        android:background="@color/colorPrimary"
-        app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
-        app:popupTheme="@style/My.PopupOverlay">
-
-        <fragment
-            android:id="@+id/wallet_balance_fragment_toolbar"
-            android:name="de.schildbach.wallet.ui.WalletBalanceToolbarFragment"
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:layout_gravity="center_vertical|right"
-            tools:layout="@layout/wallet_balance_toolbar_fragment" />
-
-    </androidx.appcompat.widget.Toolbar>
-
-    <RelativeLayout
-        android:background="@color/medium_gray"
-        android:layout_width="match_parent"
-        android:id="@+id/toolbar_bottom"
-        android:layout_height="50dp">
-
-        <TextView
-            android:id="@+id/toolbar_message"
-            android:gravity="center"
-            android:layout_centerInParent="true"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            tools:text="Synchronizing with network. 3 months behind."
-            android:textColor="@color/white"
-            app:fontFamily="@font/inter_regular"
-            android:textSize="12sp" />
-
-    </RelativeLayout>
-
-</com.google.android.material.appbar.AppBarLayout>
```

Comment:
This is some relic that's only used in sevaral two-pane layouts. Not sure if we still need those layouts (it's probably better to handle tablet UI separately), but I replaced `app_bar_main` with `app_bar_general` there for now.